src/pages/Registration/Steps/Documentation/schema.ts

```diff
@@ -38,9 +39,10 @@ export const schema = Yup.object().shape<SchemaShape<FormValues>>({
   hqCountry: Yup.object().shape<SchemaShape<CountryOption>>({
     name: requiredString,
   }),
-  endowDesignation: Yup.string()
-    .required("required")
-    .url("invalid Endowment Designation"),
+  endowDesignation: Yup.object().shape<SchemaShape<OptionType<string>>>({
```

Comment:
@abbasali1994, kindly update schema in `Admin/Charity/EditProfile/schema`

and update `src/pages/Admin/Review/Applications/__tests__/Applications.test.tsx` with the new `EndowDesignation` types

Reply:
addressed in  5587b9f86